GETTING_STARTED.md

```diff
@@ -36,13 +36,23 @@ Select a demo scenario from the [demo catalog](catalog/README.md) and copy the d
 
 Now that you've set up your local environment and chosen a demo, you can deploy your services:
 
+```console
+$ docker run -it\
+    -v $HOME/configs/:/mnt/deployer/configs/\
+    --entrypoint ruby ghcr.io/newrelic/deployer main.rb -d <demo-url>
+```
+
+Don't forget to replace `<demo-url>` with the url or local path to a deployment configuration file.
+
+If you named your user config file something other than 'user.credentials.json' use this command:
+
 ```console
 $ docker run -it\
     -v $HOME/configs/:/mnt/deployer/configs/\
     --entrypoint ruby ghcr.io/newrelic/deployer main.rb -c configs/<username>.docker.local.json -d <demo-url>
```

Comment:
I would update the path for the user config in this command, since it still follows the old `<username>.docker.local.json` pattern

Reply:
Done!